hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/sink/clustering/FlinkClusteringConfig.java

```diff
@@ -69,13 +83,14 @@ public class FlinkClusteringConfig extends Configuration {
       required = false)
   public Integer archiveMaxCommits = 30;
 
-  @Parameter(names = {"--schedule", "-sc"}, description = "Not recommended. Schedule the clustering plan in this job.\n"
-      + "There is a risk of losing data when scheduling clustering outside the writer job.\n"
-      + "Scheduling clustering in the writer job and only let this job do the clustering execution is recommended.\n"
-      + "Default is true", required = false)
-  public Boolean schedule = true;
+  @Parameter(names = {"--schedule", "-sc"}, description = "Schedule the clustering plan in this job.\n"
+      + "Default is false", required = false)
+  public Boolean schedule = false;
+
+  @Parameter(names = {"--instant-time", "-it"}, description = "Clustering Instant time")
+  public String clusteringInstantTime = null;
```

Comment:
Can we explain the use case why explicit instant time is needed for clustering execution ?

Reply:
In the scenario where users are running a scheduled execution running schedule & execute w/ LIFO instant selection + probability of a job failure. 

Users can re-execute a failed clustering operation by specifying `--instant-time`.

Reply:
Let's rename `--instant-time` to `--instant`

Reply:
From `HoodieClusteringJob.java`
```
@Parameter(names = {"--instant-time", "-it"}, description = "Clustering Instant time, only used when set --mode execute. "
        + "If the instant time is not provided with --mode execute, "
        + "the earliest scheduled clustering instant time is used by default. "
        + "When set \"--mode scheduleAndExecute\" this instant-time will be ignored.")
    public String clusteringInstantTime = null;
```

Should we standardise the parameter? Given that the Spark parameter is using `--instant-time`, we should ensure that both of them are the same to avoid confusion.

Reply:
Fine, we can keep that if spark already name it like this.